frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -460,3 +470,65 @@ action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') do
 action.GitMachete.CheckoutFirstChildAction.text=Git Machete: Checkout First Child Branch
 action.GitMachete.CheckoutFirstChildAction.description=Checkout first child branch (''{0}'')
 action.GitMachete.CheckoutFirstChildAction.undefined.branch-name=Branch (''{0}'') does not have any children in the branch layout
+
+# Traverse
+
+action.GitMachete.BaseTraverseAction.description=Traverse
+action.GitMachete.BaseTraverseAction.description.empty-layout=Cannot traverse an empty branch layout
+
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.title=Traverse Info
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.text.HTML=<html>\
+  This action triggers an interactive walk over the branches and suggests actions (pushes/pulls/rebases) \
+  necessary to achieve both child-to-parent and local-to-remote synchronization.<br><br>\
+  Do you want to proceed?\
+  </html>
+
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.text.HTML=\
+  <html>\
+    <p>The branch <b>{0}</b> is diverged from remote <b>{1}</b>, and it is <b>{2}</b> remote.</p> \
+    <p>Please choose the preferred action for syncing it:</p>\
+  </html>
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.title=Resolve Divergence from Remote
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.newer-than-remote=older than
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.older-than-remote=newer than
+
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option=Do not sync
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option.comment=Note: the remote and local changes will remain intact; traverse will continue
```

Comment:
Maybe:
```suggestion
action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option.comment=Note: the remote and local branches will remain intact; traverse will continue
```

Reply:
Also:

<img width="827" alt="image" src="https://user-images.githubusercontent.com/3383210/205505501-00826385-98af-4cad-98d9-9880a09cf0d6.png">

but if that wrapping is hard to fix, let's leave as is